.gitbook/3-protocol-devs/1-dep-specs/2-data-deal.md

```diff
@@ -0,0 +1,117 @@
+# Data Deal
+
+- Status: Draft
+- Crated: 2023-01-03
+- Modified: 2023-01-03
+- Authors
+  - Youngjoon Lee <yjlee@medibloc.org>
+  - Gyuguen Jang <gyuguen.jang@medibloc.org>
+  - Hansol Lee <hansol@medibloc.org>
+  - Myongsik Gong <myongsik_gong@medibloc.org>
+  - Inchul Song <icsong@medibloc.org>
+
+
+## Synopsis
+
+This document defines a data deal which is a contract for data collecting and pay for provision in [DEP](../../1-users/3-data-exchange/0-about-dep.md).
+Data consumers create data deal by specifying what type of data and how much they want, and how much they are willing to pay for it.
```

Comment:
```suggestion
Data consumers create data deals by specifying what type of data and how much they want, and how much they are willing to pay for it.
```

It's better to use the plural form when you say about a general concept of something.